interna/crowdfund/views.py

```diff
@@ -40,7 +41,10 @@ def post(self, request, *args, **kwargs):
 
         msg = 'Dein Beitrag wurde gespeichert, vielen Dank!'
         messages.add_message(request, messages.INFO, msg)
-        return HttpResponseRedirect(reverse('crowdfund:detail', kwargs={'pk': self.object.pk}))
+        response = HttpResponseRedirect(reverse('crowdfund:detail', kwargs={'pk': self.object.pk}))
+        response.set_cookie('last_pledge_name', form.cleaned_data.get('name'),
+                max_age=3600 * 24 * 265)
```

Comment:
Well, a year doesn't have 256 days, but it doesn't have 265 either :laughing:

Reply:
OMG how fucked up am I :rofl:

Reply:
BUT WHAT ABOUT LEAP SECONDS?